### DISCUSSION TITLE
Bot Command "Order" Target Token Options

### DISCUSSION THREAD

**Discussion by Sampsonne** [trickerer/Trinity-Bots#714] [Ideas]
I'm working on ways to have a smoother 5 man group experience on difficult content that requires smart use of crowd control.  Ideally, the mage would reliably keep a specific target sheeped, the warlock would banish, etc.  In a live server multiplayer setting, the tank would mark raid target icons and communicate which icon means which action and count on the group to get it done.  I'm working on ways to have a similar experience and by default mage does ok without much coaching, but direct control would be more reliable  I'm thinking of a scenario like the first heroic dungeons you run in tbc.  Good cc is critical.  Maybe I'm just inexperienced and this is achievable via target mask settings or some other way, but I've had little success.

My idea is extending the target token options of the order command to include raid target icons.  I could write a macro to have one mage sheep moon and a second mage sheep triangle as an example.  This could have huge benefit in raid scenarios as well,not just 5 man.  Thanks for all you've done already.  It's amazing and fun as is.  Great work.

**Comment by trickerer** [comment]
Yeah, that's a good idea. Looking at existing code seems like it won't even require much to accomplish.
Order system is currently a little buggy i.e. sometimes orders stay active (don't get aborted at fail or smth). Didn't get to debugging it yet.

**Comment by trickerer** [comment]
https://github.com/trickerer/Trinity-Bots/commit/31efedb71662ac07ab83b212c3f5e8b1179c782c
